xml/src/main/java/org/ehcache/xml/service/DefaultCacheEventListenerConfigurationParser.java

```diff
@@ -78,6 +79,9 @@ public CacheType unparseServiceConfiguration(CacheConfiguration<?, ?> cacheConfi
 
       Set<ListenersType.Listener> listeners = serviceConfigs.stream().map(serviceConfig -> {
         ListenersType.Listener listener = new ListenersType.Listener();
+        if (serviceConfig.getInstance() != null) {
+          throw new XmlConfigurationException("XML translation when instance is provided is not supported");
```

Comment:
I would like the message to tell what instance if missing. "XML translation when a CacheEventListener instance is provided in the configuration is not supported"